.github/workflows/msys2_build.yml

```diff
@@ -31,14 +25,16 @@ jobs:
   build:
     needs: changes
     if: ${{ github.repository == 'OpenVisualCloud/Media-Transport-Library' && needs.changes.outputs.changed == 'true' }}
-    runs-on: windows-latest
+    runs-on: ${{ matrix.win-ver }}
     strategy:
       fail-fast: false
       matrix:
-        sys:
-          - mingw64
-          - ucrt64
-        dpdk: [23.11, 23.07]
+        include:
+          - { sys: "mingw64", env: "x86_64", dpdk: "23.11", win-ver: "windows-2019" }
+          - { sys: "mingw64", env: "x86_64", dpdk: "23.11", win-ver: "windows-2022" }
+          # - { sys: "mingw64", env: "x86_64", dpdk: "23.11", win-ver: "windows-2025" }
+          # - { sys: "ucrt64",  env: "ucrt-x86_64", dpdk: "23.11", win-ver: "windows-2022" }
+          # - { sys: "clang64",  env: "clang-x86_64", dpdk: "23.11", win-ver: "windows-2022" }
```

Comment:
this comment is a leftover or needed ?

Reply:
Needed leftover - we should try to successfully  enable those. I had no luck with that only because some values from OS are not passed.